Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector.py

```diff
@@ -0,0 +1,279 @@
+
+import demistomock as demisto
+import urllib3
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+import traceback
+from typing import Dict, Tuple
+
+# Disable insecure warnings
+urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+''' CONSTANTS '''
+
+MAX_EVENTS_PER_REQUEST = 100
+OBJECT_KEYS = {'events': 'timestamp'}
```

Comment:
timestamp is not a good key. We need to add a demistoID to the data
we can :
Keep a counter in memory
Use a uuid
Id suggest hashing the data too but that would remove duplicates, so maybe not

Reply:
these are the only keys where i saw are mutual to all api responses

Reply:
I also think that in this case we can simply get the old context if exists, add the found event in case 200 has returned and save it to context, that way we wouldn't even need the ```OBJECT_KEYS```

Reply:
lets add UUIDs

Reply:
i think it should be a timestamp combined with a uuid4, code snippet:
```
def test_module(client: Client):
    """
    Testing we have a valid connection to Saas-Security.

    The reason we must use the get events endpoint is because Saas-Security have different scopes for each type
    of api. To verify the customer have the log access scope we must try and fetch an event.
    Saves the events in the cache for fetching events.
    """
    response = client.get_event_request()
    if response.status_code == 200:
        fetched_event = response.json()
        demisto.debug(f'fetched event in test-module: {fetched_event}')
        fetched_event['uuid'] = f'{fetched_event.get("timestamp")}{uuid.uuid4()}'
        set_to_integration_context_with_retries(context=fetched_event, object_keys=OBJECT_KEYS)
        return 'ok'
    elif response.status_code == 204:
        return 'ok'
```

The chances of it being the same is almost impossible.

Reply:
why add timestamp to uuid? Uuid is enough

Reply:
timestamp addition is just for a case where to uuid's that were created (again super low chance) will actually be identical

Reply:
ok used uuid